rules-java-diva/addon/pom.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss.windup.rules.apps</groupId>
+        <artifactId>windup-rules-java-diva-parent</artifactId>
+        <version>5.2.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>windup-rules-java-diva</artifactId>
+
+    <name>Windup Rules - Java / DiVA - Basic</name>
+
+    <dependencies>
+        <!-- Local 3rd party Dependencies -->
+        <dependency>
+            <groupId>org.jooq</groupId>
+            <artifactId>joox</artifactId>
+            <version>1.2.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.github.konveyor.tackle-diva</groupId>
+            <artifactId>diva-core</artifactId>
+            <version>0.1.1</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>com.ibm.wala</groupId>
```

Comment:
It seems like many IBM Wala dependencies are used from [`diva-core`](https://github.com/konveyor/tackle-diva/blob/0.1.1/core/build.gradle#L38-L43): it would be better if `diva-core` could provided these IBM Wala dependencies to ensure the proper version is used.

Reply:
I think windup components can omit versions either by windup-bom or parent module.  Do you think we can provide diva-bom from jitpack? (I don't think wala provides any bom...)